.travis.yml

```diff
@@ -41,28 +77,25 @@ notifications:
       on_failure: always
 
 cache:
-    apt: true
```

Comment:
Was this cache intentionally disabled?

Reply:
As per [this comment](https://github.com/travis-ci/travis-ci/issues/5876#issuecomment-207661127), apt cache doesn't really do what most people expect, and in our case, it had no effect.